### PR TITLE
[Reply] contextual page title

### PR DESCRIPTION
**Pull request by Renzo-Olivares**
Bottom app bar page title is now contextual and vanishes when going to MailView.

reply-page-title-context

### CODE REVIEW

File: lib/studies/reply/adaptive_nav.dart [hunk 1]
```diff
@@ -627,16 +627,23 @@ class _MobileNavState extends State<_MobileNav> with TickerProviderStateMixin {
...
+                        return AnimatedOpacity(
+                          opacity:
+                              _bottomDrawerVisible | onMailView ? 0.0 : 1.0,
+                          duration: const Duration(milliseconds: 350),
+                          child: Text(
+                            _currentDestination,
+                            style: Theme.of(context)
+                                .textTheme
+                                .bodyText1
+                                .copyWith(color: ReplyColors.blue50),
```
Comment (rami-a) [inline lib/studies/reply/adaptive_nav.dart]: Should this be white50 like it was before?

**Review by rami-a — APPROVED** [review]
LGTM

File: lib/studies/reply/adaptive_nav.dart (hunk 1 above)
Comment (Renzo-Olivares) [reply lib/studies/reply/adaptive_nav.dart]: Yeah I missed this.

**Review by guidezpl — APPROVED** [review]
```
  _    ___ _____ __  __ 
 | |  / __|_   _|  \/  |
 | |_| (_ | | | | |\/| |
 |____\___| |_| |_|  |_|
                        
```

**Pull request merged**